src/configure/configure.ts

```diff
@@ -465,13 +467,15 @@ class PipelineConfigurer {
                 break;
         }
 
-        let selectedResource: QuickPickItemWithData = await this.controlProvider.showQuickPick(
-            selectAppText,
-            resourceArray,
-            { placeHolder:  placeHolderText },
-            TelemetryKeys.WebAppListCount);
-
-        this.inputs.targetResource.resource = selectedResource.data;
+        if(!!this.inputs.pipelineParameters.pipelineTemplate.targetType && !!this.inputs.pipelineParameters.pipelineTemplate.targetKind) {
```

Comment:
should you add the check for target type in line 453 as existing code will create client and make bunch of rest calls

Reply:
Done